os/board/rtl8721csm/Kconfig

```diff
@@ -5,6 +5,14 @@
 
 if ARCH_BOARD_RTL8721CSM
 
+config BOOTLOADER_FIXED_PARTITIONS
+    bool "Use bootloader that supports fixed partitions"
+    depends on !USE_BP
+
+config BOOTLOADER_DYNAMIC_PARTITIONS
+    bool "Use bootloader that supports dynamic partitions"
+    depends on USE_BP
```

Comment:
There is no default. Is it ok on menuconfig? When we change CONFIG_USE_BP to y or to n, Is this changed automatically without any Kconfig syntax warning? Could you confirm this?

Reply:
Thanks, I have added the default values. Now its working as inteneded.